src/main/java/org/ideoholic/curium/model/degreedetails/dto/Degreedetails.java

```diff
@@ -1,220 +1,84 @@
 package org.ideoholic.curium.model.degreedetails.dto;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 // default package
 // Generated 5 Jul, 2018 7:53:04 PM by Hibernate Tools 5.3.0.Beta2
 
 /**
  * Degreedetails generated by hbm2java
  */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "degreedetails")
 public class Degreedetails implements java.io.Serializable {
```

Comment:
```Degreedetails.hbm.xml``` it has ```optimistic-lock="true"```
@musaibasrar - What is this for and can we emulate this in java mapping file?